dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -697,6 +1027,43 @@ def pipe(max_batch_size, input_data, device, input_layout):
     check_pipeline(generate_data(31, 13, (5, 160, 80, 3), lo=0, hi=255, dtype=np.uint8),
                    pipeline_fn=pipe, input_layout="FHWC")
 
+def test_segmentation_select_masks():
+    def get_data_source(*args, **kwargs):
+        return lambda: make_batch_select_masks(*args, **kwargs)
```

Comment:
Why the lambda, can't you call it directly?

Reply:
Done